include/zenoh-pico/api/types.h

```diff
@@ -54,25 +54,24 @@ extern "C" {
     } z_view_##name##_t;
 
 /**
- * Represents a variable-length encoding unsigned integer.
+ * Represent a variable-length encoding unsigned integer.
  *
  * It is equivalent to the size of a ``size_t``.
  */
 typedef _z_zint_t z_zint_t;
 
 /**
- * Represents an array of bytes.
```

Comment:
AFAIK in English, when the subject of a sentence is a third-person singular noun or pronoun (he, she, it, or a singular noun like "type"), the verb typically takes an 's' or 'es' at the end in the present tense.
And this comment is essentially an abbreviation for: `The type represents an array of bytes. `

Reply:
It's kind of the eternal debate between third person and imperative tense. Guess third person is more appropriate here so yeah I'll revert that